### PR TITLE
add prompt about KUBE_REPO_PREFIX

### PR DESCRIPTION
**Pull request by xilabao**
ref to https://github.com/kubernetes/kubeadm/issues/257

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/3597)

### CODE REVIEW

File: docs/admin/kubeadm.md [hunk 1]
```diff
@@ -388,6 +388,15 @@ configuration file.
 
 If `KUBE_KUBERNETES_DIR` is specified, you may need to rewrite the arguments of the kubelet.
 (e.g. --kubeconfig, --pod-manifest-path)
+If `KUBE_REPO_PREFIX` is specified, you may need to set --pod-infra-container-image which is the pause image.
+```bash
```
Comment (chenopis) [inline docs/admin/kubeadm.md]: Put an extra newline before the bash commands so that there is some space around it. Thx

**Comment by chenopis** [comment]
@xilabao We just fixed a Travis issue, so if you update the branch, the Travis CI check should now work. Thx

**Comment by xilabao** [comment]
Done. PTAL

**Comment by xilabao** [comment]
rebased.

**Comment by xilabao** [comment]
ping @chenopis

**Comment by chenopis** [comment]
I just need a quick tech review on this. @luxas can you take a look?

File: docs/admin/kubeadm.md [hunk 2]
```diff
@@ -389,6 +389,17 @@ configuration file.
 If `KUBE_KUBERNETES_DIR` is specified, you may need to rewrite the arguments of the kubelet.
 (e.g. --kubeconfig, --pod-manifest-path)
 
+If `KUBE_REPO_PREFIX` is specified, you may need to set --pod-infra-container-image which is the pause image.
```
Comment (luxas) [inline docs/admin/kubeadm.md]: suggest: 

If \`KUBE_REPO_PREFIX\` is specified, you may need to set the kubelet flag \`--pod-infra-container-image\` which specifies which pause image to use.
Defaults to \`gcr.io/google_containers/pause-${ARCH}:3.0\` where \`${ARCH}\` can be one of \`amd64\`, \`arm\`, \`arm64\`, \`ppc64le\` or \`s390x\`

**Comment by xilabao** [comment]
done. @luxas @chenopis PTAL

**Pull request merged**